packages/gatsby/src/commands/build.ts

```diff
@@ -419,6 +420,42 @@ module.exports = async function build(
   await waitMaterializePageMode
   const waitWorkerPoolEnd = Promise.all(workerPool.end())
 
+  // create scope so we don't leak state object
+  {
```

Comment:
The position of this is quite arbitrary (at the very end of the build). Any reason to not keep it there?